src/box/sql.h

```diff
@@ -417,6 +417,15 @@ func_sql_expr_has_single_arg(const struct func *base, const char *name);
 uint32_t
 sql_default_session_flags(void);
 
+/**
+ * Entrypoint for fuzzing SQL engine.
+ *
+ * @param sql UTF-8 encoded SQL statement.
+ * @param sql_len Length of @sql in bytes.
+ */
+int
+sql_fuzz(const char *sql, int bytes_count);
```

Comment:
I think you should include this prototype into
```c
#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
...
#endif
```

Reply:
Added:

```diff
--- a/src/box/sql.h
+++ b/src/box/sql.h
@@ -423,8 +423,10 @@ sql_default_session_flags(void);
  * @param sql UTF-8 encoded SQL statement.
  * @param sql_len Length of @sql in bytes.
  */
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
 int
 sql_fuzz(const char *sql, int bytes_count);
+#endif /* FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION */
 
 #if defined(__cplusplus)
 } /* extern "C" { */
```